src/md/enc/metamodelrw.cpp

```diff
@@ -8780,7 +8780,7 @@ bool FilterTable::IsTokenMarked(
         //
         return true;
     }
-    return ( (*Get(rid)) & bitMarked ? true : false);
+    return ( ((*Get(rid)) & bitMarked) ? true : false);
```

Comment:
Since you are touching this, could you please also remove the extra space after the first paren?